modules/tinymce/src/core/main/ts/api/AddOnManager.ts

```diff
@@ -242,7 +244,7 @@ function AddOnManager<T>(): AddOnManager<T> {
      *
      * @method get
      * @param {String} name Add-on to look for.
-     * @return {tinymce.Theme/tinymce.Plugin} Theme or plugin add-on instance or undefined.
+     * @return {tinymce.PluginManager} Theme or plugin add-on instance or undefined.
```

Comment:
This and the other places below aren't correct. What's returned is the plugin/theme instance, which is another reason why I think we should keep the current API pages.

Reply:
Yeah, have changed this locally, haven't pushed it up yet